src/main/java/org/primefaces/extensions/config/PrimeExtensionsEnvironment.java

```diff
@@ -59,6 +61,10 @@ public static void setCurrentInstance(PrimeExtensionsEnvironment pfeEnv, FacesCo
         }
     }
 
+    public boolean isCommonmarkAvailable() {
```

Comment:
Maybe just call this `isMarkdownAvailable`

Reply:
I opted to use the library name.